owncloudApp/src/main/java/com/owncloud/android/ui/activity/FolderPickerActivity.kt

```diff
@@ -48,19 +49,22 @@ open class FolderPickerActivity : FileActivity(),
     MainFileListFragment.FileActions {
 
     protected val mainFileListFragment: MainFileListFragment?
-        get() = supportFragmentManager.findFragmentByTag(TAG_LIST_OF_FOLDERS) as MainFileListFragment?
+       get() = supportFragmentManager.findFragmentByTag(TAG_LIST_OF_FOLDERS) as MainFileListFragment?
 
     private lateinit var pickerMode: PickerMode
 
+    private  lateinit var binding : FilesFolderPickerBinding
     override fun onCreate(savedInstanceState: Bundle?) {
         Timber.d("onCreate() start")
 
         super.onCreate(savedInstanceState)
 
+        binding = FilesFolderPickerBinding.inflate(layoutInflater)
         setContentView(R.layout.files_folder_picker)
```

Comment:
Now that we introduced ViewBinding, we can set its root to the content view

```suggestion
        setContentView(binding.root)
```

Reply:
Updated , made the changes